libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-data-manager-adapter.directive.spec.ts

```diff
@@ -190,7 +190,7 @@ describe('SkyAgGridDataManagerAdapterDirective', () => {
     agGridComponent.columnMoved.emit(columnMoved);
     agGridDataManagerFixture.detectChanges();
     await agGridDataManagerFixture.whenStable();
-    await new Promise((resolve) => setTimeout(resolve, 0));
+    await new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
What changed to require this?

Reply:
I'll remove that. This test is flaky and just randomly fails (existing issue) and that was an attempt to see if it would help - but it doesn't appear to have and I just forgot to revert that.